common/model/media-object-list.ts

```diff
@@ -0,0 +1,9 @@
+import { HTMLString } from '../services/prismic/types';
+import { ImageType } from './image';
+
+export type MediaObjectType = {
+  id: string,
+  title: string,
```

Comment:
`title` is almost definitely going to be optional, and we have no way of enforcing it in Primsic.

We should reflect this in the type to ensure we type check it.

Reply:
good point. wish they could add validation on the model.

Reply:
You;d never really want to add validation in this way, having worked on a few CMS's it just causes issues.

The reason being is that it's an anti-pattern to require the editors to have to put something in as it can hinder the editing process.

You might have a post publish check list, which is better, and [we do that in a way on the frontend](https://github.com/wellcomecollection/wellcomecollection.org/blob/master/common/views/pages/_app.js#L283).

We could probably use that pattern more extensively, but it's been such a small issue, non-issue even, that it's not been pursued.

Reply:
interesting .. thanks for that .. very useful to know about that